app/controllers/api/v1/announcements_controller.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+class Api::V1::AnnouncementsController < Api::BaseController
+  before_action -> { doorkeeper_authorize! :write, :'write:accounts' }, only: :dismiss
+  before_action :require_user!, except: :index
```

Comment:
Hm, it could be interesting to have announcements visible only to logged-in users.
Is it useful to give access to unauthenticated users, anyway?

Reply:
I thought it might be useful in the same way an instance's custom emojis are publicly available, but I'm not attached to that idea.

Reply:
I'd actually vote against that, since it may not be obvious for admins that announcements could be accessed by anonymous users, and in the case of small instances with trusted users, announcements may possibly contain sensitive information.